contributing.md

```diff
@@ -0,0 +1,11 @@
+# Contributing to the CEPC and the Positive Work Environment Community Group
+
+We welcome contributions from the community as we work to develop a positive and inclusive work environment for W3C. 
+
+If you have feedback or questions relating to the Code of Ethics and Professional Conduct or any of our other projects, please log an issue in this repository. Logging issues allows the chairs to review and raise issues in upcoming meetings. 
```

Comment:
Are contributors welcome at the meetings? If so, would it be appropriate to link to the W3C Calendar for the PWE CG?